src/index.js

```diff
@@ -1,14 +1,17 @@
 import componentsPlugin from './components'
 import directivesPlugin from './directives'
-import { installFactory, vueUse } from './utils/plugins'
+import { installFactory } from './utils/plugins'
 import { setConfig } from './utils/config'
 
+// Named export of all compoents and component group plugins
+export * from './components/index.esm'
+
+// Named export of all directives and directive group plugins
+export * from './directives/index.esm'
+
 const BootstrapVue = {
   install: installFactory({ plugins: { componentsPlugin, directivesPlugin } }),
   setConfig: setConfig
 }
 
-// Auto installation only occurs if window.Vue exists
-vueUse(BootstrapVue)
-
 export default BootstrapVue
```

Comment:
It is exported as default here

Reply:
So that `es/` build has a default export, as well as common.js and esm.js builds (for those importing the whole _kit-and-kaboodle_).